tests/containers/isolation.pm

```diff
@@ -0,0 +1,95 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: isolation
+# Summary: Test container network isolation
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal qw(select_serial_terminal select_user_serial_terminal);
+use containers::common qw(install_packages);
+use utils qw(script_retry systemctl);
+
+my $runtime;
+my $network_base = "test_internal_net";
+
+sub test_ip_version {
+    my $ip_version = shift;
+
+    my $network = "$network_base-$ip_version";
+    my $opts = $ip_version == 6 ? "--ipv6" : "";
+    assert_script_run "$runtime network create $opts --internal $network";
```

Comment:
We can move this call outside the routine and enable IPv6 by default. This doesn't need to be done for each IP stack individually I think.

Reply:
Fixed.